spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'sign_in' do
+    fixtures :users
+    it "success" do
+      post :sign_in, params: { email: "passenger1@dramancompany.com", password: "eiusmod" }
+      expect(response).to have_http_status(200)
+    end
+
+    it "incorrect credentials" do
+      post :sign_in, params: { email: "drama@dramancompany.com", password: "eiusmod" }
+      expect(response).to have_http_status(400)
+    end
+  end
```

Comment:
```suggestion
  describe '#sign_in' do
    fixtures :users
    context '유효한 이메일과 패스워드인 경우' do
      it "HTTP STATUS 200을 응답한다" do
        post :sign_in, params: { email: "passenger1@dramancompany.com", password: "eiusmod" }
        expect(response).to have_http_status(200)
      end
    end
    
    context '유효하지 않은 이메일과 패스워드인 경우' do
      it "HTTP STATUS 400을 응답한다" do
        post :sign_in, params: { email: "drama@dramancompany.com", password: "eiusmod" }
        expect(response).to have_http_status(400)
      end
    end
  end
```

이건 저희가 사용하는 컨벤션인데요,
class method인 경우에는 '.method_name'
instance method인 경우에는 '#method_name' 으로 테스트하려는 메소드명을 적어주시면 됩니다!

그 다음으로는 context '~~한 경우' 로 케이스를 적어주시고
it '~~한다' 식으로 메세지를 적어주시면 될 것 같아요!

Reply:
설명 감사합니다! 요거 여기 develop PR에서 말고 [테스트코드 PR](https://github.com/travelingaries/taxi-dispatch-backend-api/pull/2)에서 작업해서 커밋 올리겠습니다 🙆‍♂️

Reply:
@soolab 테스트코드들 리팩토링 완료해서, 새로 리뷰해주시면 감사합니다!